imap/mailbox.c

```diff
@@ -1924,6 +1926,13 @@ static int mailbox_buf_to_index_header(const char *buf, size_t len,
 
     i->changes_epoch = ntohl(*((bit32 *)(buf+OFFSET_CHANGES_EPOCH)));
 
+    if (i->minor_version < 18) goto crc;
+
+    i->quota_deleted_used = align_ntohll(buf+OFFSET_QUOTA_DELETED_USED);
+    i->quota_expunged_used = align_ntohll(buf+OFFSET_QUOTA_EXPUNGED_USED);
+
+    /* v19 rearranged fields to get modseq_t and quota_t on 8-byte boundaries */
```

Comment:
both these functions are going to have to have a fork placed earlier up so they can read the old formats correctly from the old offsets, since the constants have changed!

Reply:
This should now be fixed